scripts/azure_tags_env_variables.py

```diff
@@ -0,0 +1,16 @@
+from subprocess import Popen, PIPE
+import json
+
+raw_response = Popen(["curl", "-H", "Metadata:true", "http://169.254.169.254/metadata/instance?api-version=2017-08-01"], stdout=PIPE)
+res = raw_response.stdout.read()
+
+res_json = json.loads(res)
+res_tags = res_json['compute']['tags']
+
+file_obj = open("AzureTagsEnvVariables", "w")
```

Comment:
You can do: 

open("AzureTagsEnvVariables", "w") as file_obj:

Reply:
makes sense.  I've put this in a with block to make things cleaner.